From 3444a43a01cd4773fdcd4a75c42424916a40b751 Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Tue, 12 Nov 2019 08:56:00 -0600
Subject: [PATCH 015/120] arm64: dts: agilex: Add SysMgr to Ethernet nodes

commit 561d74759efcdfb8b7a595df0e6c861939e58241 from
https://github.com/altera-opensource/linux-socfpga.git

[backport 'commit 83d36d212c23 ("arm64: dts: agilex: Add SysMgr
				 to Ethernet nodes")']
from dinguyen/linux/for_next_v5.5_socfpga_dts_v2

Ethernet needs the System Manager to setup the low-level
interface and PHY.

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 82cfc7887c0d..3a8da29770a4 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -151,6 +151,7 @@
 			iommus = <&smmu 1>;
 			clocks = <&clkmgr AGILEX_EMAC0_CLK>;
 			clock-names = "stmmaceth";
+			altr,sysmgr-syscon = <&sysmgr 0x44 0>;
 			status = "disabled";
 		};
 
@@ -168,6 +169,7 @@
 			iommus = <&smmu 2>;
 			clocks = <&clkmgr AGILEX_EMAC1_CLK>;
 			clock-names = "stmmaceth";
+			altr,sysmgr-syscon = <&sysmgr 0x48 8>;
 			status = "disabled";
 		};
 
@@ -185,6 +187,7 @@
 			iommus = <&smmu 3>;
 			clocks = <&clkmgr AGILEX_EMAC2_CLK>;
 			clock-names = "stmmaceth";
+			altr,sysmgr-syscon = <&sysmgr 0x4c 16>;
 			status = "disabled";
 		};
 
-- 
2.31.1

